src/app/tests/suites/TestBasicInformation.yaml

```diff
@@ -53,46 +53,6 @@ tests:
           value: "XX"
 
     - label: "Read AttributeList value"
-      PICS: PICS_EVENT_LIST_ENABLED
```

Comment:
This would be easier to review if things were a bit more split up.

For example, all the PICS_EVENT_LIST_ENABLED bits are completely independent of anything else and can go in a separate PR.